packages/fxa-auth-server/fxa-oauth-server/lib/routes/token.js

```diff
@@ -141,6 +141,11 @@ const PAYLOAD_SCHEMA = Joi.object({
     is: GRANT_FXA_ASSERTION,
     otherwise: Joi.forbidden(),
   }),
+
+  ppid_seed: Joi.number()
+    .greater(-1)
+    .less(17)
```

Comment:
0..16 is an unusual range.  Do you have any analysis to support that?

Reply:
Nope, just a number pulled out of thin air.